### PR TITLE
DS-299 new brand icons

### PR DESCRIPTION
**Pull request by mikemai2awesome**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-299

## Summary

Adds a group of brand icons to the Icon component.

## Details

1. 22 new icons from branding added.
2. Certain old icons have been renamed and deprecated. The old names point to the new names in the twig template.

## How to test

Run the branch locally and go through the icon name variations doc and the new Icon Pega Branding page under Icon docs. Make sure all icons are rendering as expected.

## Release notes

The following icon names have been deprecated, please update to the new names:

1. `discussions` renamed to `pega-chat-typing`
2. `questions` renamed to `pega-chat-question`
3. `healthcare` renamed to `pega-healthcare`

### Visual changes

The following icons have slight visual updates:

1. `discussions`/`pega-chat-typing` stroke is now thinner to match new brand icons
2. `questions`/`pega-chat-question` stroke is now thinner to match new brand icons
3. `healthcare`/`pega-healthcare` is converted to the new brand icon

### CODE REVIEW

File: packages/components/bolt-icon/icon.schema.json [hunk 1]
```diff
@@ -120,6 +120,28 @@
             "more",
             "omni-channel",
             "partners",
+            "pega-cart",
```
[the remark this answers is not in the document]
Comment (mikemai2awesome) [reply packages/components/bolt-icon/icon.schema.json:123]: it kept yelling at me about schema validation, maybe that's what' i missed. let me try again.

File: packages/components/bolt-icon/src/icon.twig [hunk 2]
```diff
@@ -14,6 +14,17 @@
   {% set color = color %}
 {% endif %}
 
+{# These icons have been renamed and converted to new brand icons. #}
```
[the remark this answers is not in the document]
Comment (mikemai2awesome) [reply packages/components/bolt-icon/src/icon.twig:17]: added: https://pegadigitalit.atlassian.net/browse/DS-494

**Review by remydenton — APPROVED** [review]
Looks good.  Thanks for the release notes, I think we can take care of removing those three icons for 95% of use cases with some quick updates when we pull this into Drupal.
